core/src/main/java/io/ddf/content/SchemaHandler.java

```diff
@@ -97,68 +101,39 @@ public Schema generateSchema() throws DDFException {
   public synchronized void setFactorLevels(String columnName, Factor<?> factor) throws DDFException {
```

Comment:
The name and the param factor of type Factor<?> does not fit together. Should be setFactorLevels(List<Object> levels) ?